challenge/src/App.jsx

```diff
@@ -1,45 +1,270 @@
 import React, { Component } from "react";
 import BabyYoda from "./mandalorian/BabyYoda";
-import NumberLogger from './NumberLogger';
+import NumberLogger from "./NumberLogger";
 import strings from "./strings.json";
-import Navbar from './components/Navbar'
-import Fade from 'react-reveal/Fade'
+import Navbar from "./components/Navbar";
+import Fade from "react-reveal/Fade";
 
 class App extends Component {
-    state = {}
-    constructor(props) {
-        super(props);
-    }
-    render() {
-        console.log("Render Method is Working. Redered successfully! I think?")
+  state = {};
+  constructor(props) {
+    super(props);
+    this.state = { currentPage: 0 };
+    this.handlePageChange = this.handlePageChange.bind(this);
+  }
+
+  handlePageChange(index) {
+    this.setState({ currentPage: index });
+  }
+
+  Router(param) {
+    console.log(param.PageRoute);
+    switch (param.PageRoute) {
+      case 0:
+        return (
+          <div>
```

Comment:
breaking each page into its own component like i think you said in your email would be cool

Reply:
I went ahead and did this, the pages are now in src/components/Pages. There is a lot of reused code between each page but in a real website it would make more sense to keep each page separate